frontend/app/tests/unit/specs/components/settings/onboarding-settings.spec.ts

```diff
@@ -0,0 +1,228 @@
+import { type Wrapper, mount } from '@vue/test-utils';
+import Vuetify from 'vuetify';
+import { type Pinia } from 'pinia';
+import { beforeEach } from 'vitest';
+import OnboardingSettings from '@/components/settings/OnboardingSettings.vue';
+import { useMainStore } from '@/store/main';
+
+vi.mock('@/composables/electron-interop', () => ({
+  useInterop: vi.fn().mockReturnValue({
+    isPackaged: true,
+    restartBackend: vi.fn(),
+    config: vi.fn().mockReturnValue({
+      logDirectory: '/Users/home/rotki/logs'
+    })
+  })
+}));
+
+let saveOptions = vi.fn();
+vi.mock('@/composables/backend', async () => {
+  const mod = await vi.importActual<typeof import('@/composables/backend')>(
+    '@/composables/backend'
+  );
+  return {
+    ...mod,
+    useBackendManagement: vi.fn().mockImplementation(loaded => {
+      const mocked = mod.useBackendManagement(loaded);
+      saveOptions = vi.fn().mockImplementation(opts => {
+        mocked.saveOptions(opts);
+      });
+      return {
+        ...mocked,
+        saveOptions
+      };
+    })
+  };
+});
+
+vi.mock('@/composables/api/info/index', () => ({
+  useInfoApi: vi.fn().mockReturnValue({
+    info: vi.fn().mockReturnValue({
+      version: {
+        our_version: '1.28.0',
+        latest_version: null,
+        download_url: null
+      },
+      dataDirectory: '/Users/home/rotki/develop_data',
+      logLevel: 'DEBUG',
+      acceptDockerRisk: false,
+      backendDefaultArguments: {
+        maxLogfilesNum: 3,
+        maxSizeInMbAllLogs: 300,
+        sqliteInstructions: 5000
+      }
+    }),
```

Comment:
does this mock an API response? do you think it would be possible to use `msw` instead?